### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by singlecask** [sshuttle/sshuttle#750] [General]
I just wanted to drop a quick note to say thank you for this project!

It meets my needs (forwarding network traffic for certain subnets, and DNS, via a remote server over SSH) perfectly, unlike any other equivalent solutions that I can find. Much appreciated.

**Comment by brianmay** [comment]
Your welcome :-)

Having said that there is a lot I would like to do with this project, if only I had the time...
